index.bs

```diff
@@ -1627,6 +1664,90 @@ Releases the resources associated with the {{MLTensor}}. This method is idempote
 
 Note: Since no further operations can be enqueued using this tensor, implementations can free any additional resource allocations associated with this tensor once all previously submitted operations using it are complete.
 
+## {{MLConstantTensor}} interface ## {#api-mlconstanttensor}
+
+The {{MLConstantTensor}} interface represents a tensor which may be used as a constant to an {{MLGraph}}. The memory backing an {{MLConstantTensor}} should be allocated in an [=implementation-defined=] fashion according to the requirements of the {{MLContext}} and the {{MLOperandDescriptor}} used to create it. Operations involving the {{MLConstantTensor/[[data]]}} of an {{MLConstantTensor}} occur on the {{MLContext/[[timeline]]}} of its associated {{MLContext}}.
+
+The [=implementation-defined=] requirements of how an {{MLConstantTensor}} is allocated may include constraints such as that the memory is allocated with a particular byte alignment or in a particular memory pool.
+
+<script type=idl>
+[SecureContext, Exposed=(Window, Worker)]
+interface MLConstantTensor : MLTensor {
```

Comment:
Since `MLConstantTensor` inherits from `MLTensor`, a lot of the added spec text seems unnecessary - the internal slots should be inherited, definitions like "An MLConstantTensor's dataType..." aren't needed, etc.

Like in C++, only the behavior differences need to be called out (like virtual method overrides), and like in C++ this might require refactors in the "base class". 

Also, presumably `writeTensor()` (and `readTensor()` ?)  should throw if an `MLConstantTensor` is passed.

Reply:
>Since MLConstantTensor inherits from MLTensor, a lot of the added spec text seems unnecessary

SGTM.

> Also, presumably writeTensor() (and readTensor() ?) should throw if an MLConstantTensor is passed.

It is valid to read a constant tensor but not write - assigned static usages would throw.